app/src/main/res/values/strings.xml

```diff
@@ -1471,4 +1471,9 @@
     <!-- depcrecated: text for the firefox account onboarding card header
     The first parameter is the name of the app (e.g. Firefox Preview) -->
     <string name="onboarding_firefox_account_header">Get the most out of %s.</string>
+
+    <!-- Deprecated: No Open Tabs Message Header -->
```

Comment:
We probably won't need to group the deprecated strings by feature, they can be lumped together. Optionally, if we want to remove deprecated strings at some point, we could group the deprecated sections by quarantine dates (e.g. strings from X date to Y date)

Reply:
I've just been sliding them all at the bottom of the file but yeah if i knew the dates that would make more sense 😆 I'll file an issue about it :)